src/main/java/com/denizenscript/depenizen/bukkit/events/crackshot/crackshotPlayerFiresProjectileEvent.java

```diff
@@ -0,0 +1,127 @@
+package com.denizenscript.depenizen.bukkit.events.crackshot;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import com.shampaggon.crackshot.events.WeaponPreShootEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+import java.util.Arrays;
+
+public class crackshotPlayerFiresProjectileEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // crackshot player fires projectile
+    //
+    // @Regex ^on crackshot player fires projectile$
+    //
+    // @Triggers just before a projectile is launched from a CrackShot weapon.
+    //
+    // @Cancellable true
+    //
+    // @Context
+    // <context.weapon_name> returns the name of the weapon about to fire.
+    // <context.bullet_spread> returns the spread of the projectiles being fired.
+    // <context.left_click> returns whether the fire was a left click.
+    // <context.shot_sounds> returns a ListTag(ElementTag) of the shot sounds.
+    //
+    // @Determine
+    // "BULLET_SPREAD:" + ElementTag(Number) to set the bullet spread.
+    // "RELOAD_SOUNDS:" + ElementTag to set the shot sounds. Use "NONE" to have no sound.
+    // https://github.com/Shampaggon/CrackShot/wiki/The-Complete-Guide-to-CrackShot#sounds
```

Comment:
`<@link url` and also put it on the same line or in a different part of the meta. Each line under `@Determine` should be one possible determination.

Reply:
This has been changed.